packages/snap/src/rpc.ts

```diff
@@ -29,14 +29,36 @@ export async function getPublicKey(request: AntelopeRequest): Promise<string> {
   return String(await derivePublicKey(chain));
 }
 
+const MAX_TRANSACTION_LENGTH = 100_000;
```

Comment:
Would love to hear everyone's thoughts on whether or not this is a good max length.

Reply:
Probably a decent starting point, I'm not sure what a sane limit is or how big transaction data might get.